checkout.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<!--insert meta data here-->
+	<title>Adoptadog - Seattle's premiere dog adoption agency</title>
+	<link rel="stylesheet" type="text/css" href="./styles/styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/3f336ee7e9.js" crossorigin="anonymous"></script>
+</head>
+<body>
+
+	<!--insert contents here.-->
+    
+    <header id="top" class="pageheader">
+        <div class="headertext">
+        <h1><img src="images/adoptadog-logo.svg" width="400px" alt="adoptadog-logo"></h1>
+        <h2><i>Seattle's premiere dog adoption agency</i></h2>
+        </div>
+    </header>
+    <ul class="ulnav">
+        <li class="linav">
+            <a class="active" href="./index.html">Home</a>
+        </li>
+        <li class="linav">
+            <a href="./dogs.html">Dogs</a>
+        </li>
+        <li class="linav">
+            <a href="./blog.html">Blog</a>
+        </li>
+    </ul>
+
+	<main>
+        <h1>Checkout</h1>
+        <div>
```

Comment:
Very small, but I'd recommend seeing if you can use media queries to scale the size of your form. On desktop your fields take up the entire screen. On mobile this is great, but on desktop is a bit stretched out